frontend/apps/reader/modules/readermenu.lua

```diff
@@ -316,20 +323,29 @@ end
 
 function ReaderMenu:onSwipeShowMenu(ges)
     if self.activation_menu ~= "tap" and ges.direction == "south" then
+        local is_right_swipe = false
+        -- if the start position is far right
+        if ges.pos.x > 2 * Screen:getWidth()/3 then
+            is_right_swipe = true
+        end
         if G_reader_settings:nilOrTrue("show_bottom_menu") then
             self.ui:handleEvent(Event:new("ShowConfigMenu"))
         end
-        self.ui:handleEvent(Event:new("ShowReaderMenu"))
+        self.ui:handleEvent(Event:new("ShowReaderMenu",is_right_swipe))
```

Comment:
Could you add a space?

Reply:
I would also rename your `is_right_swipe` to the name it has in the onShowReaderMenu, so we know here what it does (as we already know you're catching swipe on the right side).

Reply:
I changed the mechanism to string "left, right, center" now, kindly have a look, and tell me your opinion